app/assets/javascripts/admin/bulk_order_management.js.coffee

```diff
@@ -162,6 +163,14 @@ angular.module("ofn.admin").controller "AdminOrderMgmtCtrl", [
       $scope.supplierFilter = $scope.suppliers[0].id
       $scope.orderCycleFilter = $scope.orderCycles[0].id
       $scope.quickSearch = ""
+
+    $scope.weightAdjustedPrice = (lineItem, oldValue) ->
+      if oldValue <= 0
+        oldValue = lineItem.units_variant.unit_value
+      if lineItem.unit_value <= 0
+        lineItem.unit_value = lineItem.units_variant.unit_value
+      lineItem.price = lineItem.price * lineItem.unit_value / oldValue
+      #$scope.bulk_order_form.line_item.price.$setViewValue($scope.bulk_order_form.line_item.price.$viewValue)
 ]
 
```

Comment:
We dowant to make sure old value isn't zero.
If it is it means the weight has never been adjusted on the line item. So if it is we can safely take the default from the product variant.
A migration would do the same thing as this code, but updating all at once. This way we only populate the field if the unit_value is different from the product variant. I feel this is a valid solution as a line-items unit_value will only occasionally be modified from the variant. Does that sound reasonable?